### ISSUE TITLE
several typos in the docs result in misrendering

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7638]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7638|
|Type|enhancement|
|Created|2014-09-15 14:23:34Z|
|Branch|https://github.com/twisted/twisted/tree/cleanup-docs-typos-7638|

These are lack of spaces between text and "code sections" (tildes).

Attachments:

* [t7638.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e85/e8590c7bc5cf17088d645587cc519e8bcff90e7a/c44d13605aa0a99c8bb79102605e8c1fe703cbe7.diff) (10013 bytes) - added by Alex on 2014-09-15 14:23:57Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7638 7638
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_cleanup_docs_typos_7638 branches/cleanup-docs-typos-7638
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1410791014195380 1410791014195380
changetime__1410865610999153 1410865610999153
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Alex set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [43098]) Branching to cleanup-docs-typos-7638.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl set status to `closed`|
|-|-|

(In [43101]) Merging cleanup-docs-typos-7638: Clean up some broken documentation links

Author: alex
Reviewer: hawkowl
Fixes: [#7638](https://github.com/twisted/twisted/issues/7638)
